### ISSUE TITLE
contacts page structure and roles

### ISSUE THREAD

**Issue by molinaro-m** [ivoa/ivoa-web#2]
In porting to markdown the (former) HTML pages the "about/contacts" one needs some attention.

A list of things to consider are:
 - the document coordinator is also the contact point for web content, i.e. web admin
 - "technical leads" can be misleading, i.e. confused with web admins
 - the "2 columns" structure currently on the page mixes up boards and duties

**Comment by molinaro-m** [comment]
the contact page is translated (basic, PR #86), and its content reflects current roles
it can be improved, but that is content-wise, not md translation

**Comment by JeremyMcCormick** [comment]
@molinaro-m Was this resolved when you ported this page?

**Comment by molinaro-m** [comment]
no, this relates to what we discussed F2F, i.e. information management that gets reflected directly on the web pages.
Lets leave this issue opened but not urgent.
